packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -38,7 +44,7 @@ import TradingPlatformIcon from '../Assets/svgs/trading-platform';
 export type TCFDPasswordFormValues = { password: string };
 
 type TExtendedDetailsOfEachMT5Loginid = Omit<DetailsOfEachMT5Loginid, 'market_type'> & {
```

Comment:
```suggestion
type TExtendedDetailsOfEachMT5Loginid = Omit<DetailsOfEachMT5Loginid, 'market_type'> & {
```

Why did you guys omit the `market_type` and then redeclaring it? Best to create a universal market type that has all the market types and reuse it everywhere. Then if you need only some you can use the `Extract<T>` / or `Exclude<T>` depending on your needs

```javascript
type Deez = 'synthetic' | 'financial' | 'gaming' | 'all'; // Base type with all
type Gotem = Extract<Deez, 'gaming'>; // Only 'gaming'
type Nit = Exclude<Deez, 'all'>; //  'synthetic' | 'financial' | 'gaming' | 
```

Reply:
The thing is that the type is coming from @deriv/api-types so thats why it is omitted and new market_types are added